pkg/pubsub/adapter/converters/storage.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2019 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package converters
+
+import (
+	"context"
+	"fmt"
+
+	cloudevents "github.com/cloudevents/sdk-go"
+	"github.com/cloudevents/sdk-go/pkg/cloudevents/transport"
+	cepubsub "github.com/cloudevents/sdk-go/pkg/cloudevents/transport/pubsub"
+	"knative.dev/pkg/logging"
+)
+
+func ConvertStorage(ctx context.Context, m transport.Message, err error) (*cloudevents.Event, error) {
```

Comment:
I would change this signature to 

```go
func ConvertStorage(ctx context.Context, msg *cepubsub.Message, err error) (*cloudevents.Event, error) {
```

because you already have access to the `cepubsub.Message`: `if msg, ok := m.(*cepubsub.Message); ok {`